src/VisualStudio/CSharp/Test/ProjectSystemShim/LegacyProject/CSharpReferencesTests.cs

```diff
@@ -14,6 +14,7 @@ namespace Roslyn.VisualStudio.CSharp.UnitTests.ProjectSystemShim.LegacyProject
     [UseExportProvider]
     public class CSharpReferenceTests
     {
+        /*
```

Comment:
>/* [](start = 8, length = 2)

TODO?

Reply:
Ah, these are all redundant now, just going to delete.